server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -1470,173 +1469,191 @@ public ClusterState getClusterStateForManifest(
         String localNodeId,
         boolean includeEphemeral
     ) throws IOException {
-        ClusterState stateFromCache = remoteClusterStateCache.getState(clusterName, manifest);
-        if (stateFromCache != null) {
-            return stateFromCache;
-        }
+        try {
+            ClusterState stateFromCache = remoteClusterStateCache.getState(clusterName, manifest);
+            if (stateFromCache != null) {
+                return stateFromCache;
+            }
 
-        final ClusterState clusterState;
-        final long startTimeNanos = relativeTimeNanosSupplier.getAsLong();
-        if (manifest.onOrAfterCodecVersion(CODEC_V2)) {
-            clusterState = readClusterStateInParallel(
-                ClusterState.builder(new ClusterName(clusterName)).build(),
-                manifest,
-                manifest.getClusterUUID(),
-                localNodeId,
-                manifest.getIndices(),
-                manifest.getCustomMetadataMap(),
-                manifest.getCoordinationMetadata() != null,
-                manifest.getSettingsMetadata() != null,
-                includeEphemeral && manifest.getTransientSettingsMetadata() != null,
-                manifest.getTemplatesMetadata() != null,
-                includeEphemeral && manifest.getDiscoveryNodesMetadata() != null,
-                includeEphemeral && manifest.getClusterBlocksMetadata() != null,
-                includeEphemeral ? manifest.getIndicesRouting() : emptyList(),
-                includeEphemeral && manifest.getHashesOfConsistentSettings() != null,
-                includeEphemeral ? manifest.getClusterStateCustomMap() : emptyMap(),
-                false,
-                includeEphemeral
-            );
+            final ClusterState clusterState;
+            final long startTimeNanos = relativeTimeNanosSupplier.getAsLong();
+            if (manifest.onOrAfterCodecVersion(CODEC_V2)) {
+                clusterState = readClusterStateInParallel(
```

Comment:
nit: avoid unintended formatting

Reply:
This is because this whole block is added in a try-catch block, causing one level indent